ocs_ci/utility/rosa.py

```diff
@@ -231,6 +236,18 @@ def install_odf_addon(cluster):
         cmd = cmd + f" --notification-email-1 {notification_email_1}"
     if notification_email_2:
         cmd = cmd + f" --notification-email-2 {notification_email_2}"
+    if cluster_type.lower() == "provider":
+        public_key = config.AUTH.get("managed_service", {}).get("public_key", "")
+        if not public_key:
+            raise ConfigurationError(
+                "Public key for Managed Service not defined.\n"
+                "Expected following configuration in auth.yaml file:\n"
+                "managed_service:\n"
+                '  private_key: "..."\n'
+                '  public_key: "..."'
+            )
+        public_key_only = remove_header_footer_from_key(public_key)
+        cmd = cmd + f" --onboarding-validation-key " + '"' + f"{public_key_only}" + '"'
```

Comment:
Just suggestion, you can use `cmd += f" ...."` instead of `cmd = cmd + f"..."` (it is little bit shorter and clearer when reviewing the code).

Reply:
changed